box_unreal_wrapper.py

```diff
@@ -0,0 +1,83 @@
+from ue5env import UE5EnvWrapper
+import ue5env
+import sys
+from boxnavigator import Action, BoxNavigatorBase
+from math import degrees, radians
+from box import Box, Pt
+
+sys.path.append(
+    "C:/Users/simon/OneDrive/Documents/ArcsLab/ArcLabPrograms/OldenborgTraining/DataCollection"
+)
```

Comment:
This should not be hard-coded.

Reply:
Unsure how to set up the program where you don't have to hard code in the paths to the other repositories